terraform/hub/manifets/secretStorev2-prod.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: spoke-prod
+  namespace: argocd
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: aws-secrets-manager
+  refreshInterval: "1m"
+  target:
+    name: spoke-prod
+    creationPolicy: Owner
+    template:
+      engineVersion: v2
+      templateFrom:
+      - target: Annotations
+        literal: "{{ .metadata }}"
+      - target: Labels
+        literal: "{{ .addons }}"
+      - target: Labels
+        literal: "argocd.argoproj.io/secret-type: cluster"
+      - target: Labels
+        literal: "cluster_name: '{{ .cluster_name }}'"
+      - target: Labels
+        literal: "environment: '{{ .environment  }}'"
+      data:
+        config: "{{ .config  }}"
+        name:   "{{ .cluster_name  }}"
+        server: "{{ .server  }}"
+  dataFrom:
+    - extract:
+        key: hub-cluster/spoke-prod
```

Comment:
Where comes from this key ?

Reply:
Those are created from the spoke cluster - creating a secret that the hub is pulling